File: absurdityAppForJava/src/main/java/forOlderJava/absurdityAppForJava/global/exception/ErrorCode.java
package forOlderJava.absurdityAppForJava.global.exception;

import org.springframework.http.HttpStatus;

public enum ErrorCode {

    ORDER_NOT_FOUND("Can not Cancel the Order");

    private final String message;

    ErrorCode(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }
}

File: ForOlder/src/main/kotlin/absurdity/ForOlder/common/exception/ErrorCode.kt
package absurdity.ForOlder.common.exception

import org.springframework.http.HttpStatus

enum class ErrorCode (
    val status: HttpStatus,
    val serial: String,
    val message: String
) {
    ORDER_NOT_FOUND(HttpStatus.BAD_REQUEST, "ERROR_01", "해당 심부름이 없습니다, 확인 해보세요"),
    CANNOT_CANCEL_ORDER(HttpStatus.BAD_REQUEST, "ERROR_02", "안쓸듯")
}